cfme/control/explorer/conditions.py

```diff
@@ -271,6 +273,9 @@ def create(self, condition_class, description, expression=None, scope=None, note
         view.add_button.click()
         view = condition.create_view(ConditionDetailsView, wait="10s")
         view.flash.assert_success_message('Condition "{}" was added'.format(condition.description))
+        if BZ(1683697, forced_streams=["5.9", "5.10"]).blocks:
```

Comment:
I don't think you need the forced_streams here, it should block without it.

Reply:
@jawatts testing this, I'm seeing that it's necessary.